iib/workers/tasks/build_merge_index_image.py

```diff
@@ -109,20 +113,32 @@ def _add_bundles_missing_in_source(
         )
 
     _opm_index_add(
-        base_dir, missing_bundle_paths, binary_image, overwrite_target_index_token,
+        base_dir=base_dir,
+        bundles=missing_bundle_paths,
+        binary_image=binary_image,
+        from_index=source_from_index,
```

Comment:
The problem was here: from_index=source_from_index was removed in previous release.